URL import docs (#70)
crate/cloud-reference: docs/overview.rst

first-time users, an organization called "My organization" is automatically 
created upon first login.

> To see list of all the organizations your account is associated with, go to 

Reviewer: ```suggestion
To see a list of all the organizations you have acesss to, go to 

```

If you're interested in this feature, fill out the provided form by clicking
"Request Private Link".

.. _overview-cluster-data:

Data
----

In the Data tab, users can conveniently import data from a URL.

.. image:: _assets/img/cluster-data-tab.png
   :alt: Cloud Console cluster data tab

To import data, simply fill out the URL, name of the table which will be

> created and populated with your data, data format, and compression.

Reviewer 2: ```suggestion
created and populated with your data, data format, and whether it is compressed.
```

> Following data formats are supported:

Reviewer 2: ```suggestion
The following data formats are supported:
```

- CSV
- JSON
- Parquet

> Gzip compressed files are also supported.

Reviewer 2: Might also be worth mentioning that:
```
A table will be created for you if it does not exist. Please note that in this case the importer will attempt to guess the data types of each column, and might do so sub-optimally. 
```

.. NOTE::

    - CSV files must have a header, otherwise the first row will be used as
      headers.

>     - For JSON documents, each line must be a JSON document (arrays not
>       supported)
>     - For JSON documents, only "document-per-line" is supported -> 
>       https://jsonlines.org/

Reviewer 2: Maybe also make jsonlines.org a link?
```suggestion
    - For JSON files, only "document-per-line" is supported, as defined in https://jsonlines.org/
    - Additionally for JSON files, each line must be a JSON document (arrays not
      supported)
```
